API/Controllers/ReaderController.cs

```diff
@@ -394,6 +394,8 @@ public async Task<ActionResult<bool>> MarkChaptersUntilAsRead(int seriesId, floa
             var user = await _unitOfWork.UserRepository.GetUserByUsernameAsync(User.GetUsername(), AppUserIncludes.Progress);
             user.Progresses ??= new List<AppUserProgress>();
 
+            if (chapterNumber == 0.0f) return true;
```

Comment:
Please put a comment explaining why Tachiyomi might send a 0.0 and why we should not process on it.